go/src/koding/klient/build.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+REPO_PATH=$(git rev-parse --show-toplevel)
+CHANNEL=${1:-}
+VERSION=${2:-}
+
+export GOBIN="${REPO_PATH}"
```

Comment:
No, $REPO_PATH is ok - ./build.sh outputs binaries to top level repo dir, ./deploy.sh reads them and uploads to S3. Didn't want to use go/bin to not overwrite existing binaries.